ej/models.py

```diff
@@ -8,6 +8,8 @@
 class Problem(db.Entity):
     title = Required(str, unique=True)
     time_limit = Required(int)
+    custom_check = Required(bool)
```

Comment:
When a custom check script is necessary, the field would be filled with the script code. When it is not, the field would be nil. This is why I think we don't need the bool field.